mephisto/tools/scripts.py

```diff
@@ -221,7 +224,7 @@ def augment_config_from_db(script_cfg: DictConfig, db: "MephistoDB") -> DictConf
     return script_cfg
 
 
-def build_custom_bundle(custom_src_dir, run_config: DictConfig):
+def build_custom_bundle(custom_src_dir, **kwargs: Union[str, bool]):
```

Comment:
kwargs is type Union[str, bool] because the parameter value can be a string(post_install_script) or bool(force_rebuild) currently.

Reply:
Shouldn't you be able to just write this as:
```
def build_custom_bundle(custom_src_dir, force_rebuild: Optional[bool] = None, post_install_script: Optional[str] = None)
```

Reply:
Yeah, I can do it like that, I was just following Pratik's suggestion here: https://github.com/facebookresearch/Mephisto/pull/791#discussion_r914119245

Reply:
@Etesam913, you'd still be using kwargs with @JackUrb's suggestion above by naming the arguments in the list

more info: https://treyhunner.com/2018/04/keyword-arguments-in-python/

Reply:
Oh so kwargs just refers to passing the arguments with keywords as opposed to using the **kwargs parameter. I got it 👍